assignments/objects.js

```diff
@@ -19,27 +19,80 @@ const example = {
 
 // Write your intern objects here:
 
+const employeeOne = {
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
+  }
+
+const employeeTwo = {
+  "id": 2,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M"
+  
+  }
+const employeeThree = {
+  "id": 3,
+  "name": "Keven",
+  "email": "kmummery2@wikimedia.org",
+  "gender": "M"
+  
+  }
+
+const employeeFour = {
+  "id": 4,
+  "name": "Gannie",
+  "email": "gmartinson3@illinois.edu",
+  "gender": "M"
+  
+  }
+
+  const employeeFive = {
+  "id": 5,
+  "name": "Antonietta",
+  "email": "adaine5@samsung.com",
+  "gender": "F"
+  
+  }
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
 
+console.log(employeeOne["name"]);
+
 // Kennan's ID
 
+console.log(employeeTwo["id"]);
+
 // Keven's email
 
+console.log(employeeThree["email"]);
+
 // Gannie's name
 
+console.log(employeeFour["name"]);
+
 // Antonietta's Gender
 
+console.log(employeeFive["gender"]);
+
+
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
 
+console.log("Hello, my name is " + (employeeTwo["name"])+"!");
+
+
 // Antonietta loves math, give her the ability to multiply two numbers together and return the product. Use the console.log provided as a hint.
 //console.log(antonietta.multiplyNums(3,4));
 
```

Comment:
So, the second part of this challenge wants you to build upon what you've done above, so same instance, you call: antonietta.multiplyNums = function (two params goes in here) {
  (Your block of code goes in here);
}
(Then console.log the object method and invoke it with two inputs for the two parameters.)